### DISCUSSION TITLE
7.2.0

### DISCUSSION THREAD

**Discussion by antonmedv** [google/zx#590] [General]
<h3 align="center">🐚 zx v7.2.0 release! 🎉</h3>
<h3 align="center">A tool for writing better scripts</h3>
<p align="center">
  <img width="600" alt="zx" src="https://user-images.githubusercontent.com/141232/173627713-94247047-a683-4e43-bf6a-aaf3f1f4a4c4.png">
</p>

In this release:
- Helpers [retry()](https://github.com/google/zx#retry) & [spinner()](https://github.com/google/zx#spinner) now available in **zx** without the experimental flag.
- Added support for `~~~` blocks in [markdown](https://github.com/google/zx#markdown-scripts) scripts.
- Updated npm dependencies.
- Added support for **ssh** commands via [webpod](https://github.com/webpod/webpod).

PS: [Plan](https://github.com/google/zx/issues/589) for the next **v8** release.

<hr /><em>This discussion was created from the release <a href='https://github.com/google/zx/releases/tag/7.2.0'>7.2.0</a>.</em>

**Comment by PATROMO** [comment]
Is there a way to see the fired commands with webpod in terminal? (Like with $?)

**Reply by antonmedv** [reply]
You can try with env var WEBPOD_DEBUG=true.

Will add more output options for ssh command.
